source/conf.py

```diff
@@ -0,0 +1,154 @@
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+sys.path.insert(0, str(Path(__file__).parent.resolve()))
+needs_sphinx = "6.2"
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Cantera"
+copyright = "2023, Cantera Developers"
+author = "Cantera Developers"
+html_title = "Cantera"
+
+# The full version, including alpha/beta/rc tags
+version = "2.6"
+release = "2.6.0"
+
+source_suffix = {
+    ".rst": "restructuredtext",
+    ".txt": "restructuredtext",
+    ".md": "markdown",
+}
+
+# Print warnings into the built HTML files
+keep_warnings = True
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions: list[str] = [
+    "sphinx_design",
+    "myst_parser",
+    "sphinx.ext.duration",
+    "sphinx.ext.mathjax",
+    "sphinxcontrib.mermaid",
+    "ablog",
+    "sphinx.ext.intersphinx",
+]
+
+myst_enable_extensions = ["colon_fence", "deflist", "attrs_block", "attrs_inline"]
+myst_url_schemes = {
+    "http": None,
+    "https": None,
+    "mailto": {"url": "mailto:{{path}}", "title": "{{path}}"},
+}
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ["files"]
+html_extra_path = ["files/license/license.txt", "files/surveys/"]
+
+root_doc = "index"
+
+# -- Internationalization ---------------------------------------------------
+language = "en"
+
+intersphinx_mapping = {
+    'stable': ('https://testing.cantera.org/stable', None),
+    'dev': ('https://testing.cantera.org/dev', None),
+    'ct30': ('https://testing.cantera.org/3.0/sphinx/html', None),
+    'ct26': ('https://testing.cantera.org/2.6/sphinx/html', None),
+}
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "pydata_sphinx_theme"
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ["_static"]
+html_logo = "_static/img/cantera-logo.png"
+html_favicon = "_static/img/favicon.png"
+html_css_files = ["css/custom.css"]
+html_js_files = ["js/copybutton.js"]
+html_sidebars: dict[str, list | list[str]] = {
+    "index": [],
+    "news/**": [
+          'ablog/postcard.html', 'ablog/recentposts.html',
+          'ablog/tagcloud.html']
```

Comment:
Are these valid links? The `ablog` prefix doesn't seem right, although these could be generated pages.

Reply:
These refer to templates within `ablog`: See https://ablog.readthedocs.io/en/stable/manual/templates-themes.html#sidebars and https://github.com/sunpy/ablog/tree/main/src/ablog/templates/ablog.